modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -175,5 +114,6 @@ Configurations applied to the `user-workload-monitoring-config` `ConfigMap` obje
 +
 [WARNING]
 ====
-When changes are saved to a monitoring config map, the pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+When you save changes to a monitoring config map, the pods and other resources in the project might be redeployed.
```

Comment:
Should this be `ConfigMap` here like in other places?

Reply:
@rh-tokeefe - We can use either one: "config map" or "`ConfigMap` object" -- so either one is correct. When I refer to specific edits to a config map, I'll usually say "`ConfigMap` object"